lms/static/sass/views/_learner-profile.scss

```diff
@@ -23,6 +23,76 @@
         width: ($baseline*5);
     }
 
+    .profile-image-field {
+        @include float(left);
+
+        button {
+            background: transparent !important;
+            border: none !important;
+            padding: 0;
+            -webkit-tap-highlight-color: transparent;
+        }
+
+        .u-field-image {
+            padding-top: 0;
+        }
+
+        .image-wrapper {
+            width: $profile-image-dimension;
+            position: relative;
+
+            .image-frame {
+                position: relative;
+                width: 120px;
```

Comment:
@muhammad-ammar Can you use the `$profile-image-dimension` for all these `120px` values?